dockerfiles/Dockerfile.training

```diff
@@ -142,6 +142,9 @@ RUN cd /stage && git clone https://github.com/microsoft/onnxruntime.git &&\
         --skip_tests &&\
     pip install build/${BUILD_CONFIG}/dist/*.whl
 
+# Install AzureML support and commonly used packages.
+RUN pip install azureml-defaults transformers==2.11.0 msgpack tensorboardX==1.8 tensorboard==2.3.0
```

Comment:
Last time in the meeting, we reached conclusion that this file is going to be ORT base image with just enough dependencies to build ORT and AzureML will own the the image with AzureML dependencies in onnxruntime-training-examples repo. Is that still the case? @manashgoswami

Reply:
ok to add these packages here. the ownership transition will be from next release.

Reply:
When possible, please move "azureml-defaults transformers==2.11.0 msgpack tensorboardX==1.8 tensorboard==2.3.0" to requirements.txt, so that our IP scan tool can auto detect the components then you don't need to manually update cgmanifest.json.